library/ecjpake.c

```diff
@@ -651,6 +673,10 @@ int mbedtls_ecjpake_write_round_two( mbedtls_ecjpake_context *ctx,
     unsigned char *p = buf;
     const unsigned char *end = buf + len;
     size_t ec_len;
+    ECJPAKE_VALIDATE_RET( ctx   != NULL );
```

Comment:
Not a blocker: please leave a blank line after the block of declarations.

Reply:
Fixing.